detector/aws_elasticache_cluster_default_parameter_group.go

```diff
@@ -7,7 +7,15 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsElasticacheClusterDefaultParameterGroup(issues *[]*issue.Issue) {
+type AwsElastiCacheClusterDefaultParameterGroupDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateAwsElastiCacheClusterDefaultParameterGroupDetector() *AwsElastiCacheClusterDefaultParameterGroupDetector {
```

Comment:
__[golint]__
* [golint] exported method Detector.CreateAwsElastiCacheClusterDefaultParameterGroupDetector should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->